doc/help/MERGE.md

```diff
@@ -0,0 +1,44 @@
+kdb-merge(1) -- Three-way merge of KeySets.
+================================
+## DESCRIPTION
+Does a three-way merge between keysets.  
+On success the resulting keyset will be saved to mergepath.  
+On unresolved conflicts nothing will be changed.  
+
+## USAGE
```

Comment:
Yes I thought that would be enough at first. But sometimes specific explanations of individual flags could be useful.

So additional text provided (in addition whats in --help anyway) can be useful.